File: 2 semestre/Laboratório de Desenvolvimento de Algoritmos/Lista de Exercícios 1/src/Ex10.java
//Importando Scanner
import java.util.Scanner;

public class Ex10{
	
	public static void main(String[] args){
		
	//Instanciando Scanner
	Scanner ent = new Scanner(System.in);
	
	//vars
	double lado, area;
	
	System.out.println("Área de um quadradado\n\n");
	
	//Entrada	
	System.out.print("Insira o tamanho do lado do quadrado: ");
	lado = ent.nextDouble();
	
	//Processamento
	area = Math.pow(lado, 2);
	
	//Saída
	System.out.printf("Área = %.2f", area);
	
	}
}

File: 2 semestre/Técnica de Desenvolvimento de Algoritmos/Listas de Exercicios/Lista 1/ex10.c
#include <stdio.h>
#include <math.h>

main()
{
    //vars
    float area, lado;

    printf("\t\t\tArea de um quadrado\n\n");

    //Entrada
    printf("Insira o tamanho do lado do quadrado: ");
    scanf("%f",&lado);

    //Processamento
    area = pow( lado, 2);

    //Saída
    printf("Area = %.2f", area);

}
